src/styles/style-helpers.js

```diff
@@ -53,6 +53,13 @@ export const textKilo = createTypeHelper('text', 'kilo');
 export const textMega = createTypeHelper('text', 'mega');
 export const textGiga = createTypeHelper('text', 'giga');
 
+/**
+ * SVGs
+ */
+
+export const svgKilo = size('16px');
```

Comment:
Can this maybe use the variable from the theme? Probably call it with the theme as argument. The function then picks the according spacing from the theme.

Reply:
I don't think this is actually a spacing though. Technically it's an SVG size, and not tied semantically to spacing IMO.